.travis.yml

```diff
@@ -110,7 +110,7 @@ cache:
 
 before_install:
    - export SOURCEDIR=$PWD
-   # select the right C++ compiler version (gcc8) -- note that update-alternatives is not available on MacOS
+   # select the right C++ compiler version (gcc6) -- note that update-alternatives is not available on MacOS
```

Comment:
Is this comment update correct? Tests seem to be using GCC8 consistently.

Reply:
@lekshmideepu: gentle ping from Hackathon Headquarters!

Reply:
You are right. Frankly speaking, I don't remember changing this line at all :stuck_out_tongue_winking_eye: